test/parallel/test-tls-connect-given-socket.js

```diff
@@ -23,16 +23,16 @@
 const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
+const fixtures = require('../common/fixtures');
 
 const assert = require('assert');
 const tls = require('tls');
 const net = require('net');
 const fs = require('fs');
```

Comment:
`fs` is no longer used, can you please remove this line?

Reply:
Its removed.